text/0022-adopt-encointer-runtime.md

```diff
@@ -0,0 +1,55 @@
+# RFC-0022: Adopt Encointer Runtime
+
+|                 |                                                       |
+| --------------- |-------------------------------------------------------|
+| **Start Date**  | Aug 22nd 2023                                         |
+| **Description** | merge Encointer runtime into fellowship runtimes repo |
+| **Authors**     | @brenzi                                               |
```

Comment:
I think a legal name would be good, otherwise maybe the legal entity of Encointer also works.